src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js

```diff
@@ -114,153 +74,129 @@ const testSecondaryTwo = createTestConfig(
           cy.get('.usa-button-primary').click();
         });
       },
-      'primary-3': ({ afterHook }) => {
+      'primary-caregiver/home-address': ({ afterHook }) => {
         afterHook(() => {
-          cy.fillPage();
-          cy.get('#root_primaryAddress_autofill')
-            .shadow()
-            .find('label')
-            .click();
-          cy.get('.usa-button-primary').click();
+          cy.get('@testData').then(data => {
+            const fieldName = 'primaryAddress';
+            const fieldData = data[fieldName];
+            fillAddressWebComponentPattern(fieldName, fieldData);
+            cy.injectAxeThenAxeCheck();
+            goToNextPage();
+          });
+        });
+      },
+      'secondary-caregiver/home-address': ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            const fieldName = 'view:secondaryOneHomeSameAsMailingAddress';
+            const fieldData = data[fieldName];
+            selectCheckboxWebComponent('caregiverAddress_autofill', true);
+            selectYesNoWebComponent(fieldName, fieldData);
+            goToNextPage();
+          });
+        });
+      },
+      'secondary-caregiver/mailing-address': ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            const fieldName = 'secondaryOneMailingAddress';
+            const fieldData = data[fieldName];
+            fillAddressWebComponentPattern(fieldName, fieldData);
+            cy.injectAxeThenAxeCheck();
+            goToNextPage();
+          });
+        });
+      },
+      'additional-secondary-caregiver/home-address': ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            const fieldName = 'secondaryTwoAddress';
+            const fieldData = data[fieldName];
+            fillAddressWebComponentPattern(fieldName, fieldData);
+            cy.injectAxeThenAxeCheck();
+            goToNextPage();
+          });
         });
       },
       'review-and-submit': () => {
         cy.get('@testKey').then(testKey => {
+          const parties = {
+            veteran: 'Micky Mouse',
+            primary: 'Mini Mouse',
+            secondaryOne: 'George Geef Goofus II',
+            secondaryTwo: 'Donald Duck',
+          };
+
           switch (testKey) {
             case 'secondaryOneOnly':
-              // Check veteran content && sign
```

Comment:
Lets not use Cypress tests to check content rendering. We can do that through unit testing.